test/spec/faros_event_spec.sh

```diff
@@ -1,28 +1,88 @@
 Describe 'faros_event.sh'
-  Include lib/faros_event_test.sh
   Describe 'faros_event.sh deployment'
     It 'Constructs correct deployment event'
+        deployment_event_test() {
+          echo $(
+            FAROS_DEPLOYMENT="<deployment_uid>" \
+            FAROS_START_TIME=10 \
+            FAROS_END_TIME=10 \
+            ../faros_event.sh deployment -k "<api_key>" \
+            --app "<app_name>" \
+            --ci_org "<ci_organization>" \
+            --ci_source "<ci_source>" \
+            --deployment_status Success \
+            --deployment_env QA \
+            --pipeline "<ci_pipeline>" \
+            --build "<build_uid>" \
+            --dry_run \
+            --no_format)
+        }
```

Comment:
We didn't need `faros_event_test.sh` so I just moved it all into a single file so that it was easier to see that call and response.